Backends/CLX-fb/package.lisp

```diff
@@ -17,7 +17,6 @@
         #:clim-clx
```

Comment:
Please change the commit message to something like "port-lookup-mirror: do not use internal protocol in backends" since the protocol is not removed. I also don't get the "obsolete function" part.

Please mention `pixmap-mirror`.

Reply:
fixed